node/plugin/cmd/main.go

```diff
@@ -117,6 +124,21 @@ func pluginOps(ctx *cli.Context) {
 		}
 	}
 
+	if config.Operation == plugin.OperationListQuorums {
+		operatorAddress, err := tx.OperatorIDToAddress(context.Background(), operatorID)
```

Comment:
maybe we should just use this to get address for all cases and not rely on private key?